app/assets/stylesheets/base/reset.scss

```diff
@@ -0,0 +1,137 @@
+// TODO: uncomment when we're ready!
```

Comment:
This whole reset file is the only file that could potentially affect visuals of DEV as of right now.  It basically changes sizing of few elements (because of global line-height) but fortunately it is NOT breaking these elements.